decidim-comments/config/locales/ca.yml

```diff
@@ -0,0 +1,22 @@
+ca:
+  decidim:
+    components:
+      add_comment_form:
+        title: "Deixa el teu comentari"
+        form:
+          body:
+            label: "Comentari"
+            placeholder: "Què opines d'aquesta proposta?"
```

Comment:
This is wrong, comments could be attached to any type of resource, not just proposals.